ext/compressors/zstd/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+# See the file LICENSE for redistribution information.
```

Comment:
This doesn't make much sense to me? Why do I need to review LICENSE for files in the public domain?

Reply:
Oh good catch!
Sorry, I didn't realize the licensing was different in the extensions. Should the license header match whats in the corresponding source e.g. for zstd:
```
* This is free and unencumbered software released into the public domain.                                                                              
*                                                                                                                                                      
* Anyone is free to copy, modify, publish, use, compile, sell, or                                                                                      
* distribute this software, either in source code form or as a compiled                                                                                
* binary, for any purpose, commercial or non-commercial, and by any                                                                                    
* means.                                                                                                                                               
*                                                                                                                                                      
 ....
```

Reply:
I've updated the `ext` CMake license headers with the public domain header. Let me know if this looks good :)